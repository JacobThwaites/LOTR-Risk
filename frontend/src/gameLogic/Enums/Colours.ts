export enum Colour {
    GREEN = 'green',
    YELLOW = 'yellow',
    RED = 'red',
    BLACK = 'black',
}

