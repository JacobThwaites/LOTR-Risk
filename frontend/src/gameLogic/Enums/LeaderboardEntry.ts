export interface LeaderboardEntry {
    colour: string,
    areasControlled: number,
    totalUnits: number,
}
