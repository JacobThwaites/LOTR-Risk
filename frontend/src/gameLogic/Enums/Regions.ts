import { Region } from '../Models/Region';
import { Areas } from './Areas';

const eriadorAreas = [Areas.FORLINDON, Areas.MITHLOND, Areas.HARLINDON, Areas.TOWER_HILLS, Areas.EVENDIM_HILLS, Areas.LUNE_VALLEY, Areas.THE_SHIRE];
const arnorAreas = [Areas.FORODWAITH, Areas.ANGMAR, Areas.EASTERN_ANGMAR, Areas.BORDERLANDS, Areas.NORTH_DOWNS, Areas.RHUDAUR, Areas.FORNOST, Areas.WEATHER_HILLS, Areas.BUCKLAND, Areas.OLD_FOREST, Areas.SOUTH_DOWNS];
const rhunAreas = [Areas.NORTH_RHUN, Areas.WITHERED_HEATH, Areas.ESGAROTH, Areas.SOUTH_RHUN];
const mirkwoodAreas = [Areas.CARROCK, Areas.NORTH_MIRKWOOD, Areas.ANDUIN_VALLEY, Areas.EASTERN_MIRKWOOD, Areas.SOUTH_MIRKWOOD];
const rohanAreas = [Areas.EREGION, Areas.MINHIRIATH, Areas.DUNLAND, Areas.ENEDWAITH, Areas.FANGORN, Areas.WEST_ROHAN, Areas.GAP_OF_ROHAN];
const rhovanionAreas = [Areas.MORIA, Areas.GLADDEN_FIELDS, Areas.LORIEN, Areas.EMYN_MUIL, Areas.BROWNLANDS, Areas.RHUN_HILLS, Areas.THE_WOLD, Areas.DEAD_MARSHES];
const gondorAreas = [Areas.DRUWAITH_IAUR, Areas.ANDRAST, Areas.ANFALAS, Areas.VALE_OF_ERECH, Areas.LAMEDON, Areas.BELFALAS, Areas.LEBENNIN, Areas.MINAS_TIRITH, Areas.ITHILIEN, Areas.SOUTH_ITHILIEN];
const mordorAreas = [Areas.UDUN_VALE, Areas.MOUNT_DOOM, Areas.BARAD_DUR, Areas.MINAS_MORGUL, Areas.GORGOROTH, Areas.NURN];
const haradwaithAreas = [Areas.HARONDOR, Areas.UMBAR, Areas.DEEP_HARAD, Areas.HARAD, Areas.NEAR_HARAD, Areas.KHAND];

interface ListOfRegions { 
    [key: string]: Region
}

export const Regions: ListOfRegions = {
    ERIADOR: new Region('eriador', eriadorAreas, 3),
    ARNOR: new Region('arnor', arnorAreas, 7),
    RHUN: new Region('rhun', rhunAreas, 2),
    MIRKWOOD: new Region('mirkwood', mirkwoodAreas, 4),
    ROHAN: new Region('rohan', rohanAreas, 4),
    ROHVANION: new Region('rhovanion', rhovanionAreas, 4),
    GONDOR: new Region('gondor', gondorAreas, 7),
    MORDOR: new Region('mordor', mordorAreas, 2),
    HARADWAITH: new Region('haradwaith', haradwaithAreas, 2),
}