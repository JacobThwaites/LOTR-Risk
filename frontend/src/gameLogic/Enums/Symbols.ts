export enum Symbol {
    ARCHER = "ARCHER",
    CAVALRY = "CAVALRY",
    EAGLE = "EAGLE",
    WILD_CARD = "WILD_CARD"
}
