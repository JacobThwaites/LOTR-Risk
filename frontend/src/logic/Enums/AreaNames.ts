export enum AreaName {
    FORLINDON = 'Forlindon',
    MITHLOND = 'Mithlond',
    LUNE_VALLEY = 'LuneValley',
    EVENDIM_HILLS = 'EvendimHills',
    TOWER_HILLS = 'TowerHills',
    THE_SHIRE = 'TheShire',
    HARLINDON = 'Harlindon',
    FORODWAITH = 'Forodwaith',
    ANGMAR = 'Angmar',
    EASTERN_ANGMAR = 'EasternAngmar',
    BORDERLANDS = 'Borderlands',
    NORTH_DOWNS = 'NorthDowns',
    RHUDAUR = 'Rhudaur',
    FORNOST = 'Fornost',
    WEATHER_HILLS = 'WeatherHills',
    OLD_FOREST = 'OldForest',
    BUCKLAND = 'Buckland',
    SOUTH_DOWNS = 'SouthDowns',
    CARROCK = 'Carrock',
    NORTH_MIRKWOOD = 'NorthMirkwood',
    EASTERN_MIRKWOOD = 'EasternMirkwood',
    SOUTH_MIRKWOOD = 'SouthMirkwood',
    ANDUIN_VALLEY = 'AnduinValley',
    NORTH_RHUN = 'NorthRhun',
    WITHERED_HEATH = 'WitheredHeath',
    SOUTH_RHUN = 'SouthRhun',
    ESGAROTH = 'Esgaroth',
    MINHIRIATH = 'Minhiriath',
    EREGION = 'Eregion',
    DUNLAND = 'Dunland',
    ENEDWAITH = 'Enedwaith',
    FANGORN = 'Fangorn',
    WEST_ROHAN = 'WestRohan',
    GAP_OF_ROHAN = 'GapOfRohan',
    MORIA = 'Moria',
    GLADDEN_FIELDS = 'GladdenFields',
    LORIEN = 'Lorien',
    THE_WOLD = 'TheWold',
    EMYN_MUIL = 'EmynMuil',
    BROWNLANDS = 'BrownLands',
    RHUN_HILLS = 'RhunHills',
    DEAD_MARSHES = 'DeadMarshes',
    DRUWAITH_IAUR = 'DruwaithIaur', 
    ANDRAST = 'Andrast',
    ANFALAS = 'Anfalas',
    VALE_OF_ERECH = 'ValeOfErech',
    LAMEDON = 'Lamedon',
    BELFALAS = 'Belfalas',
    LEBENNIN = 'Lebennin',
    MINAS_TIRITH = 'MinasTirith',
    ITHILIEN = 'Ithilien',
    SOUTH_ITHILIEN = 'SouthIthilien',
    UDUN_VALE = 'UdunVale',
    MOUNT_DOOM = 'MountDoom',
    BARAD_DUR = 'BaradDur',
    MINAS_MORGUL = 'MinasMorgul',
    GORGOROTH = 'Gorgoroth',
    NURN = 'Nurn',
    HARONDOR = 'Harondor',
    UMBAR = 'Umbar',
    DEEP_HARAD = 'DeepHarad',
    HARAD = 'Harad',
    NEAR_HARAD = 'NearHarad',
    KHAND = 'Khand',
}