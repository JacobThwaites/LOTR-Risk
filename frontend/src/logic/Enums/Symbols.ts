export enum Symbol {
    ELVEN_ARCHER = 1,
    DARK_RIDER = 3,
    EAGLE = 5,
}
