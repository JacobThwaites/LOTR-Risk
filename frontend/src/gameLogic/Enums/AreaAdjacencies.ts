import { AreaName } from './AreaNames';

interface IAdjacencies {
    [key: string]: Array<AreaName>
}

export const Adjacencies: IAdjacencies = {
}

Adjacencies[AreaName.FORLINDON] = [AreaName.MITHLOND];
Adjacencies[AreaName.MITHLOND] = [AreaName.FORLINDON, AreaName.LUNE_VALLEY, AreaName.HARLINDON, AreaName.TOWER_HILLS];
Adjacencies[AreaName.HARLINDON] = [AreaName.MITHLOND];
Adjacencies[AreaName.LUNE_VALLEY] = [AreaName.MITHLOND, AreaName.EVENDIM_HILLS, AreaName.BORDERLANDS, AreaName.TOWER_HILLS];
Adjacencies[AreaName.EVENDIM_HILLS] = [AreaName.LUNE_VALLEY, AreaName.TOWER_HILLS, AreaName.BORDERLANDS, AreaName.NORTH_DOWNS];
Adjacencies[AreaName.TOWER_HILLS] = [AreaName.MITHLOND, AreaName.EVENDIM_HILLS, AreaName.THE_SHIRE, AreaName.LUNE_VALLEY];
Adjacencies[AreaName.THE_SHIRE] = [AreaName.TOWER_HILLS, AreaName.BUCKLAND];
Adjacencies[AreaName.FORODWAITH] = [AreaName.BORDERLANDS, AreaName.ANGMAR, AreaName.EASTERN_ANGMAR, AreaName.NORTH_RHUN, AreaName.WITHERED_HEATH];
Adjacencies[AreaName.BORDERLANDS] = [AreaName.LUNE_VALLEY, AreaName.EVENDIM_HILLS, AreaName.NORTH_DOWNS, AreaName.ANGMAR, AreaName.FORODWAITH, AreaName.WEATHER_HILLS, AreaName.FORNOST];
Adjacencies[AreaName.ANGMAR] = [AreaName.FORODWAITH, AreaName.BORDERLANDS];
Adjacencies[AreaName.NORTH_DOWNS] = [AreaName.BORDERLANDS, AreaName.FORNOST];
Adjacencies[AreaName.FORNOST] = [AreaName.NORTH_DOWNS, AreaName.WEATHER_HILLS, AreaName.OLD_FOREST, AreaName.BUCKLAND, AreaName.BORDERLANDS];
Adjacencies[AreaName.WEATHER_HILLS] = [AreaName.FORNOST, AreaName.BORDERLANDS, AreaName.OLD_FOREST, AreaName.RHUDAUR, AreaName.SOUTH_DOWNS];
Adjacencies[AreaName.EASTERN_ANGMAR] = [AreaName.FORODWAITH, AreaName.CARROCK];
Adjacencies[AreaName.RHUDAUR] = [AreaName.WEATHER_HILLS, AreaName.CARROCK, AreaName.EREGION];
Adjacencies[AreaName.OLD_FOREST] = [AreaName.FORNOST, AreaName.WEATHER_HILLS, AreaName.BUCKLAND, AreaName.SOUTH_DOWNS];
Adjacencies[AreaName.BUCKLAND] = [AreaName.THE_SHIRE, AreaName.FORNOST, AreaName.OLD_FOREST, AreaName.SOUTH_DOWNS];
Adjacencies[AreaName.SOUTH_DOWNS] = [AreaName.BUCKLAND, AreaName.OLD_FOREST, AreaName.MINHIRIATH];
Adjacencies[AreaName.NORTH_RHUN] = [AreaName.FORODWAITH, AreaName.WITHERED_HEATH, AreaName.SOUTH_RHUN];
Adjacencies[AreaName.WITHERED_HEATH] = [AreaName.FORODWAITH, AreaName.NORTH_RHUN, AreaName.ESGAROTH];
Adjacencies[AreaName.ESGAROTH] = [AreaName.WITHERED_HEATH, AreaName.NORTH_MIRKWOOD];
Adjacencies[AreaName.MINHIRIATH] = [AreaName.SOUTH_DOWNS, AreaName.DUNLAND, AreaName.ENEDWAITH];
Adjacencies[AreaName.SOUTH_RHUN] = [AreaName.NORTH_RHUN, AreaName.BROWNLANDS];
Adjacencies[AreaName.CARROCK] = [AreaName.EASTERN_ANGMAR, AreaName.NORTH_MIRKWOOD, AreaName.ANDUIN_VALLEY, AreaName.RHUDAUR];
Adjacencies[AreaName.NORTH_MIRKWOOD] = [AreaName.CARROCK, AreaName.EASTERN_MIRKWOOD, AreaName.ESGAROTH];
Adjacencies[AreaName.ANDUIN_VALLEY] = [AreaName.CARROCK, AreaName.EASTERN_MIRKWOOD, AreaName.SOUTH_MIRKWOOD, AreaName.GLADDEN_FIELDS];
Adjacencies[AreaName.EASTERN_MIRKWOOD] = [AreaName.NORTH_MIRKWOOD, AreaName.ANDUIN_VALLEY, AreaName.SOUTH_MIRKWOOD, AreaName.BROWNLANDS];
Adjacencies[AreaName.SOUTH_MIRKWOOD] = [AreaName.ANDUIN_VALLEY, AreaName.EASTERN_MIRKWOOD, AreaName.BROWNLANDS, AreaName.EMYN_MUIL, AreaName.BROWNLANDS];
Adjacencies[AreaName.EREGION] = [AreaName.RHUDAUR, AreaName.MORIA, AreaName.DUNLAND];
Adjacencies[AreaName.DUNLAND] = [AreaName.EREGION, AreaName.MINHIRIATH, AreaName.ENEDWAITH];
Adjacencies[AreaName.ENEDWAITH] = [AreaName.MINHIRIATH, AreaName.DUNLAND, AreaName.GAP_OF_ROHAN];
Adjacencies[AreaName.WEST_ROHAN] = [AreaName.GAP_OF_ROHAN, AreaName.DRUWAITH_IAUR];
Adjacencies[AreaName.GAP_OF_ROHAN] = [AreaName.ENEDWAITH, AreaName.WEST_ROHAN, AreaName.FANGORN, AreaName.THE_WOLD, AreaName.MINAS_TIRITH];
Adjacencies[AreaName.FANGORN] = [AreaName.LORIEN, AreaName.THE_WOLD, AreaName.GAP_OF_ROHAN];
Adjacencies[AreaName.MORIA] = [AreaName.EREGION, AreaName.GLADDEN_FIELDS];
Adjacencies[AreaName.GLADDEN_FIELDS] = [AreaName.MORIA, AreaName.ANDUIN_VALLEY, AreaName.LORIEN];
Adjacencies[AreaName.LORIEN] = [AreaName.GLADDEN_FIELDS, AreaName.FANGORN, AreaName.THE_WOLD];
Adjacencies[AreaName.THE_WOLD] = [AreaName.LORIEN, AreaName.FANGORN, AreaName.EMYN_MUIL, AreaName.GAP_OF_ROHAN];
Adjacencies[AreaName.EMYN_MUIL] = [AreaName.ANDUIN_VALLEY, AreaName.SOUTH_MIRKWOOD, AreaName.BROWNLANDS, AreaName.DEAD_MARSHES, AreaName.THE_WOLD];
Adjacencies[AreaName.BROWNLANDS] = [AreaName.SOUTH_MIRKWOOD, AreaName.SOUTH_RHUN, AreaName.EMYN_MUIL, AreaName.RHUN_HILLS, AreaName.DEAD_MARSHES];
Adjacencies[AreaName.RHUN_HILLS] = [AreaName.BROWNLANDS];
Adjacencies[AreaName.DEAD_MARSHES] = [AreaName.EMYN_MUIL, AreaName.BROWNLANDS, AreaName.UDUN_VALE, AreaName.ITHILIEN];
Adjacencies[AreaName.DRUWAITH_IAUR] = [AreaName.WEST_ROHAN, AreaName.ANFALAS];
Adjacencies[AreaName.ANDRAST] = [AreaName.ANFALAS];
Adjacencies[AreaName.ANFALAS] = [AreaName.ANDRAST, AreaName.DRUWAITH_IAUR, AreaName.VALE_OF_ERECH];
Adjacencies[AreaName.VALE_OF_ERECH] = [AreaName.ANFALAS, AreaName.LAMEDON];
Adjacencies[AreaName.LAMEDON] = [AreaName.VALE_OF_ERECH, AreaName.BELFALAS, AreaName.LEBENNIN];
Adjacencies[AreaName.BELFALAS] = [AreaName.LAMEDON, AreaName.LEBENNIN];
Adjacencies[AreaName.LEBENNIN] = [AreaName.LAMEDON, AreaName.BELFALAS, AreaName.MINAS_TIRITH];
Adjacencies[AreaName.MINAS_TIRITH] = [AreaName.GAP_OF_ROHAN, AreaName.LEBENNIN, AreaName.ITHILIEN];
Adjacencies[AreaName.ITHILIEN] = [AreaName.DEAD_MARSHES, AreaName.SOUTH_ITHILIEN, AreaName.MINAS_MORGUL, AreaName.MINAS_TIRITH];
Adjacencies[AreaName.SOUTH_ITHILIEN] = [AreaName.ITHILIEN, AreaName.HARONDOR];
Adjacencies[AreaName.UDUN_VALE] = [AreaName.DEAD_MARSHES, AreaName.MOUNT_DOOM];
Adjacencies[AreaName.MOUNT_DOOM] = [AreaName.UDUN_VALE, AreaName.BARAD_DUR, AreaName.GORGOROTH];
Adjacencies[AreaName.BARAD_DUR] = [AreaName.MOUNT_DOOM, AreaName.GORGOROTH];
Adjacencies[AreaName.MINAS_MORGUL] = [AreaName.GORGOROTH, AreaName.ITHILIEN];
Adjacencies[AreaName.GORGOROTH] = [AreaName.MOUNT_DOOM, AreaName.BARAD_DUR, AreaName.MINAS_MORGUL, AreaName.NURN];
Adjacencies[AreaName.NURN] = [AreaName.GORGOROTH];
Adjacencies[AreaName.HARONDOR] = [AreaName.SOUTH_ITHILIEN, AreaName.HARAD];
Adjacencies[AreaName.UMBAR] = [AreaName.DEEP_HARAD];
Adjacencies[AreaName.DEEP_HARAD] = [AreaName.UMBAR, AreaName.HARAD];
Adjacencies[AreaName.HARAD] = [AreaName.DEEP_HARAD, AreaName.HARONDOR, AreaName.NEAR_HARAD];
Adjacencies[AreaName.NEAR_HARAD] = [AreaName.HARAD, AreaName.KHAND];
Adjacencies[AreaName.KHAND] = [AreaName.NEAR_HARAD];