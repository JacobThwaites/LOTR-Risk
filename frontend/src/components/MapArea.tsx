import React from "react";
import Circle from './common/Circle';
import areaDetails from "./svgPaths/AreaDetails";
import { AreaName } from "../gameLogic/Enums/AreaNames";

type Props = {
  className: string,
  areaName: string,
  onClick: any,
  isRendered: boolean
}

export default function MapArea(props: Props) {
  const areaDetail = areaDetails[props.areaName as AreaName];

  return (
    <>
      <path
        className={'area ' + props.className}
        id={props.areaName}
        d={areaDetail.path}
        onClick={props.onClick}
        />
      {props.isRendered &&
        <Circle 
          centroid={areaDetail.centroid} 
          colour={areaDetail.colour!}
          units={areaDetail.units}
        />
      }
    </>
  );
}