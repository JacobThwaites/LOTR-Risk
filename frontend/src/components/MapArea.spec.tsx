import '@testing-library/jest-dom'
import React from 'react';
import { render } from '@testing-library/react'
import MapArea from './MapArea';
import { AreaName } from '../gameLogic/Enums/AreaNames';

it('renders MapArea correctly', async () => {
    const { container } = render(
        <MapArea 
            className={'asdf'}
            areaName={AreaName.ANDRAST}
            onClick={() => {}}
            isRendered={true}
        />
    );
    expect(container).toMatchSnapshot();
});

it('renders circle with area units after isRendered prop is true', async () => {
    let isRendered = false;

    const { container, rerender } = render(
        <MapArea 
            className={'asdf'}
            areaName={AreaName.ANDRAST}
            onClick={() => {}}
            isRendered={isRendered}
        />
    );

    let areaUnits = container.getElementsByClassName('areaUnits');
    expect(areaUnits).toHaveLength(0);

    isRendered = true;

    rerender(
        <MapArea 
            className={'asdf'}
            areaName={AreaName.ANDRAST}
            onClick={() => {}}
            isRendered={isRendered}
        />
    );

    areaUnits = container.getElementsByClassName('areaUnits');
    expect(areaUnits).toHaveLength(1);
});
