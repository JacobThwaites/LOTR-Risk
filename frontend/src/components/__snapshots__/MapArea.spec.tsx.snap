// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MapArea correctly 1`] = `
<div>
  <path
    class="area asdf"
    d="m290.11 1500.8c-4.6729-0.4934-9.3458 0.1624-14.019 5.8411-4.8027 6.2148-9.6054 9.2294-14.408 8.1776-0.94916 10.222-2.7467 20.024-11.293 26.48-3.1935 8.7399-6.0468 17.43-29.984 29.206-6.2904 6.2206-13.628 17.258-17.523 12.461-6.2305 9.5021-12.461 4.6929-18.692 3.5047-2.2881 4.5686-6.9356 8.8001-12.85 12.85-3.3845 5.4517-7.6286 10.903-3.5047 16.355 10.962-5.926 22.661-5.5889 34.657-2.7259 20.208-18.485 34.147-5.6269 49.455 0.3895 4.2621-2.58 8.5582-0.64 12.85 0.7788 5.6431-3.3903 11.569-11.026 16.745-7.3988 10.405-2.7721 27.959-29.412 45.171-6.6199 12.531-5.5762 23.899 1.6476 35.436 7.0093 8.1866 0.2949 15.696-5.2832 25.312 7.3988 3.8935-4.4586 8.4427-6.7315 10.903-15.966 2.5042-2.9854 1.8968-5.9709 1.947-8.9563-0.81875-3.1224-1.2079-6.3676 1.5576-10.514 0.40911-6.7424-3.1434-11.221-6.2305-15.966-2.7363-7.4822-4.333-15.666-0.38941-27.259 4.1203-5.3543-0.60387-9.9045-1.5576-14.798 0.0994-6.8248-1.7988-12.378-7.3988-15.576-4.0268 1.7448-10.346 3.2349-10.125 5.4517-8.8031 0.4472-17.878-1.2804-25.701 7.0094-9.138 0.2836-11.441 5.2993-14.019 10.125-12.738-2.6101-19.045-1.4022-15.187 5.8411-12.373-0.9009-19.218 1.336-12.461 11.293-6.4786 0.5039-14.548 0.5534-7.0093 5.0623-0.66607 6.0758-6.771 8.3445-12.461 10.904-4.1858-0.8751-6.8755 2.7381-10.125 4.6729l-7.0094 6.6199c-8.7194-5.0198-8.0348-3.2003-5.4517 0-6.4901 4.2662-12.98 5.7201-19.47 2.7259-12.461 0.3894-12.461 2.8556 0 7.3987-9.6167 4.3624-19.459 5.11-29.595 1.1682 1.8172-5.4517 6.4902-8.567 14.019-9.3458 1.1733-6.6734 9.2335-10.395 17.523-14.019-1.9722-12.485 4.9126-6.2726 7.3988-9.3458-0.85035-4.3596 2.9862-9.1453 7.7882-14.019-5.7049-3.8941 1.4033-7.7882 5.4517-11.682 5.4898-6.7156 11.525-12.476 19.081-15.576-2.5948-2.9855 0.53984-5.9709 1.1682-8.9564z"
    id="ANDRAST"
  />
  <circle
    cx="344.01829528808594"
    cy="1558.1712646484375"
    data-testid="circle"
    r="30"
    stroke="black"
    stroke-width="3"
  />
  <text
    class="areaUnits"
    dy=".3em"
    stroke="#eee"
    stroke-width="2px"
    text-anchor="middle"
    x="344.01829528808594"
    y="1558.1712646484375"
  >
    1
  </text>
</div>
`;
